pom.xml

```diff
@@ -1484,6 +1484,9 @@
                     <artifactId>xml-apis</artifactId>
                 </exclusion>
             </exclusions>
+
+            <scope>system</scope><systemPath>${project.basedir}/lib/openlcb.jar</systemPath>
```

Comment:
Should the tag "systemPart" be on it's own line?

Reply:
The full line is the bit of 'pom.xml magic' that causes the local openlcb.jar to be used.  It should (probably) be removed when we go back to using a released version. So I put it all on one line, separated from what's around it, to try to make that clearer.  I'll add a comment to that effect.

Reply:
Thanks. Could you add a generic note at the top of pom.xml about this, with this as an example? It would be good to have this documented if it is needed for other things as well.